cla-backend/cla/models/docusign_models.py

```
@@ -1346,14 +1360,16 @@ def populate_sign_url(self, signature, callback_url=None,
                     cla.log.debug(f'{fn} - {sig_type} - '
                                   f'loading user by reference id: {signature.get_signature_reference_id()}')
                     user.load(signature.get_signature_reference_id())
+                    provider_type = (signature.get_signature_return_url_type() or '').lower()
+                    resolved_user_email = resolve_individual_user_email(user, provider_type, preferred_email)
                     cla.log.debug(f'{fn} - {sig_type} - loaded user by '
                                   f'id: {user.get_user_id()}, '
                                   f'name: {user.get_user_name()}, '
-                                  f'email: {user.get_user_email()}')
+                                  f'email: {resolved_user_email}')
                     if not user.get_user_name() is None:
```

Ruff (0.15.6)

[error] Test for object identity should be `is not`

Convert to `is not`

(E714)